inbox/react/components/preferences.mdx

```diff
@@ -33,4 +33,46 @@ function Novu() {
 
 ### Filtering preferences
 
-You can filter the preferences visible by the user by specifying the `preferenceFilter` on the `Inbox` component, learn more about it [here](/inbox/react/components/inbox#filter-visible-preferences)
\ No newline at end of file
+You can filter the preferences visible by the user by specifying the `preferenceFilter` on the `Inbox` component, learn more about it [here](/inbox/react/components/inbox#filter-visible-preferences)
+
+### Hide global preferences
+Global subscriber preferences are shown by default, it can hidden by using below code in global css file
+```css
+.nv-workflowContainer:first-child {
+  display: none;
+}
+```
+
+### Hide specific channel preferences
```

Comment:
@jainpawan21 we should promote the Appearance Prop API over CSS, or ideally demonstrate how to do this both using the Appearance Prop and the CSS in a tabbed UI in Mintlify.